test/Compatibility/CountableRange.swift

```diff
@@ -0,0 +1,11 @@
+// RUN: %target-typecheck-verify-swift
+
+struct RequiresComparable<T: Comparable> { }
+
+extension CountableRange { // expected-warning{{'CountableRange' is deprecated: renamed to 'Range'}}
```

Comment:
Can you test 'extension DictionaryIndex' too?

Reply:
Sure!